### PR TITLE
WIP: remove LA

### PR DESCRIPTION
**Pull request by CJ-Wright**
This needs the list of feedstocks to move and may need to edit the
`meta.yaml` if the script is used there.

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Ping @regro/auto-tick

**Comment by ocefpaf** [comment]
@CJ-Wright looks like the conda-build PR to implement this was simpler than we though thanks to its well designed filter files utility. See https://github.com/conda/conda-build/pull/3102/files

If @msarahan accepts that as the `default` behavior let's work on a new `conda-build` release for this script instead of the manual removal here.

File: conda_forge_tick/migrators.xsh [hunk 1]
```diff
@@ -719,3 +719,37 @@ class Pinning(Migrator):
...
+
+    def filter(self, attrs: dict):
+        if attrs['name'] in self.LA_FEEDSTOCKS and not super().filter(attrs):
+            return False
+        else:
+            return True
+
+    def migrate(self, recipe_dir, attrs, **kwargs):
+        with indir(recipe_dir):
+            for build_name in ['build' + ext for ext in ['.sh', '.bat']]:
```
Comment (scopatz) [inline conda_forge_tick/migrators.xsh]: We don't actually want to test `*.bat` files, or add this `LA_STR` to them

**Comment by CJ-Wright** [comment]
conda build does this now :tada: thank you @msarahan and @ocefpaf

**Pull request closed without merging**